src/profile-logic/call-tree.js

```diff
@@ -145,12 +156,9 @@ export class CallTree {
         : _formatDecimalNumber;
 
       displayData = {
-        totalTime: `${formatNumber(
-          this._callNodeTimes.totalTime[callNodeIndex]
-        )}`,
-        totalTimePercent: `${(100 *
-          this._callNodeTimes.totalTime[callNodeIndex] /
-          this._rootTotalTime).toFixed(precision)}%`,
+        totalTime: `${formatNumber(totalTime)}`,
+        totalTimeRelative: totalTimeRelative,
```

Comment:
Adding `totalTimeRelative` here - and in the test and type definition below - was a mistake from a recent rebase. It suffices for it to be part of `getNodeData`. I will fix it later, after I've gotten some review comments.